bitnami/zookeeper/README.md

```diff
@@ -111,7 +111,7 @@ The command removes all the Kubernetes components associated with the chart and
 
 | Name                                 | Description                                                                                                                                                                                       | Value           |
 | ------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------- |
-| `extraVolumes`                       | Extra volumes                                                                                                                                                                                     | `[]`            |
+| `extraVolumes`                       | Extra volumes  (value evaluated as a template)                                                                                                                                                                                     | `[]`            |
```

Comment:
I think that you don't need to include this in the Readme. I took a look another charts like Redis and we are not including this information. So, please remove it. Thanks.